packages/devtools_app/lib/src/inspector/inspector_screen.dart

```diff
@@ -278,6 +319,40 @@ class InspectorScreenBodyState extends State<InspectorScreenBody>
           //);
         });
       }
+
+      addAutoDisposeListener(
+          inspectorController.selectedNode, _selectedNodeChanged);
```

Comment:
Instead of calling setState on the whole inspector screen, make ErrorNavigator a stateful widget that listens for changes to `inspectorController.selectedNode`. That way we won't have to rebuild the entire tree to update the error navigator

Reply:
Done! (41863a30)